src/components/home.css

```diff
@@ -12,7 +18,6 @@
   align-items: center;
   padding: 0% 3% 0% 3%;
   width: 70%;
-  margin: 1% 15% 1% 15%;
   height: 1.8rem;
 }
 
```

Comment:
**[Optional]**

- [ ] You are not following Gitflow for the review of this pull request 😨 which was a requirement of your project. Right which you have done with your project is you are merging your development branch to main branch which is a requirement of a group project but as you know that this project is solo project so here you are supposed to use Gitflow for the review 👍